src/tests/multihost/alltests/test_rfc2307bis.py

```diff
@@ -0,0 +1,102 @@
+""" Automation of proxy provider suite
+
+:requirement: IDM-SSSD-REQ : Proxy Provider
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import time
+import os
+import re
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from constants import ds_suffix, ds_instance_name
+
+
+def execute_cmd(multihost, command):
+    """ Execute command on client """
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.fixture(scope='class')
+def ldap_objects_sssd_client(multihost, request):
+    """
+        Configure /etc/pam_ldap.conf
+        Restore
+    """
+    execute_cmd(multihost, "> /etc/pam_ldap.conf")
+    execute_cmd(multihost, f"echo 'base {ds_suffix}' "
+                           f"> /etc/pam_ldap.conf")
+    execute_cmd(multihost, "echo 'pam_password md5' "
+                           ">> /etc/pam_ldap.conf")
+    execute_cmd(multihost, f"echo 'host {multihost.master[0].ip}' "
+                           f">> /etc/pam_ldap.conf")
+    execute_cmd(multihost, "echo 'tls_cacertfile "
+                           "/etc/openldap/certs/cacert.asc'"
+                           " >> /etc/pam_ldap.conf")
+
+    execute_cmd(multihost, 'systemctl restart nslcd')
+
+    def restoresssdconf():
+        """ Restore sssd.conf """
+        execute_cmd(multihost, "rm -vf /etc/pam_ldap.conf")
+
+    request.addfinalizer(restoresssdconf)
+
+
+@pytest.mark.usefixtures('setupds',
+                         'default_sssd',
+                         'sssdproxyldap',
+                         'install_nslcd',
+                         'ldap_objects_sssd_client')
+@pytest.mark.tier1_3
+class TestProxyMisc(object):
+    """
+    This is test case class for proxy provider suite
```

Comment:
'proxy provider' isn't test target here.
`sss_client` lib(s) are.

Reply:
And we need async provider here (like sssd-ldap). I'm not sure proxy provider is.

Reply:
I don't think this is the best approach.
This is a totally different test, it belongs to own suite.
I also think using proxy provider in unnecessary complication here.

Reply:
Now its fixed